buffalogs/impossible_travel/management/commands/clear_models.py

```diff
@@ -1,12 +1,34 @@
-from django.core.management.base import BaseCommand
-from impossible_travel.models import Alert, Login, TaskSettings, User
+from django.apps import apps
+from django.core.management.base import BaseCommand, CommandParser
+from impossible_travel.models import Alert, Config, Login, TaskSettings, User
 
 
 class Command(BaseCommand):
-    help = "Clear Alert, Login, User and TaskSettings models"
+    help = "Clear data on the models"
+
+    def add_arguments(self, parser: CommandParser) -> None:
+        parser.add_argument("--model", nargs="?", type=str, help="Model to clear")
+        return super().add_arguments(parser)
 
     def handle(self, *args, **options):
-        Alert.objects.all().delete()
-        Login.objects.all().delete()
-        User.objects.all().delete()
-        TaskSettings.objects.all().delete()
+        if options["model"]:
+            if options["model"] in ["Config", "config"]:
+                choice = input(self.stdout.write("Are you sure you want to delete the Configurations model? [y/N]"))
+                if choice in ["y", "Y"]:
```

Comment:
I think it's better to remove asking the user to confirm. It could be troublesome if we need to launch the script non-interactively.